src/modules/AutoinstSoftware.rb

```diff
@@ -1020,19 +1017,19 @@ def ReadHelper
       Pkg.SourceStartManager(true)
       Pkg.PkgSolve(false)
 
-      all_patterns = Pkg.ResolvableProperties("", :pattern, "")
-      @all_xpatterns = Pkg.ResolvableDependencies("", :pattern, "")
+      all_patterns = Y2Packager::Resolvable.find(kind: :pattern)
+      @all_xpatterns = Y2Packager::Resolvable.find(kind: :pattern)
       to_install_packages = install_packages
       patterns = []
 
       # FIXME: filter method but it use only side effect of filling patterns
       Builtins.filter(all_patterns) do |p|
```

Comment:
But anyway, ignoring the result of `Builtins.filter` looks really strange. Does it make even sense? It seems simple `each` should be rather used here...